src/platform/packages/shared/kbn-unified-metrics-grid/src/components/chart/hooks/use_lens_extra_actions.ts

```diff
@@ -25,12 +26,38 @@ export const useLensExtraActions = (config: UseLensExtraActions): Action[] => {
       actions.push(getCopyToDashboardAction(config.copyToDashboard.onClick));
     }
 
+    if (config.viewDetails) {
+      actions.push(getViewDetailsAction(config.viewDetails.onClick));
+    }
+
     return actions;
-  }, [config.copyToDashboard]);
+  }, [config.copyToDashboard, config.viewDetails]);
```

Comment:
We are receiving and depending on the object itself for the `useMemo` dependencies, but I see in the rest of the code that we are not memoizing either the object itself or the callback, so wouldn't this cause this memo to be redundantly re-evaluating every time the component renders?